specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing.json

```diff
@@ -4782,6 +4782,14 @@
           "type": "string",
           "readOnly": true
         },
+        "tags": {
+          "type": "object",
+          "description": "Dictionary of metadata associated with the invoice section.",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "maxItems": 1000
```

Comment:
Any reason you need to allow this many? ARM supports just 15 per resource...

Reply:
i was trying to be consistent with labels. i know API itself only supports max 50.

Reply:
Made it to 50, that's what API supports as well.